odmlui/DocumentRegistry.py

```diff
@@ -19,7 +20,8 @@ def add(self, doc):
         """
         add a document to the registry and return its unique id
         """
-        id = hash(doc)
+        # odML.Document is unhashable. Converting to string before hashing
+        id = hash(str(doc))
```

Comment:
Yeah, to make it hashable 'internally', we need to have a `__hash__()` method.
But if we do implement one, what do we put in it ? I mean, how should be the `hash` computed ? If we would again consider the author name, version, etc. in the internal hash function, then I thought that maybe just convert the `doc` into string (which already has a `__hash()__` method), and that would have most of the above mentioned information.
So, do we implement a separate hash function ?